src/main/resources/view/DarkTheme.css

```diff
@@ -147,10 +147,11 @@
 }
 
 .result-display {
-    -fx-background-color: transparent;
+    -fx-background-color: #222222;
```

Comment:
I think this property is being contradicted by `style="-fx-background-color: #333` in `ResultDisplay.fxml`.